src/content/configuration/externals.mdx

```diff
@@ -329,23 +345,23 @@ Supported types:
 
 - `'amd'`
 - `'amd-require'`
-- `'assign'`
-- `'commonjs'`
+- `'assign'` - same as `'var'`
+- [`'commonjs'`](#externalstypecommonjs)
 - `'commonjs-module'`
-- `'global'`
+- [`'global'`](#externalstypeglobal)
```

Comment:
This `global` works same as `self` in my test, could u double check it?

![](https://user-images.githubusercontent.com/1091472/183340790-df4a7b99-d3d6-414f-9983-c25063f639a2.png)

Reply:
oh it actually depends on [`output.globalObject`](https://webpack.js.org/configuration/output/#outputglobalobject)

Reply:
i've updated the relevant change